server/routerlicious/packages/lambdas/src/scribe/lambdaFactory.ts

```diff
@@ -112,7 +113,15 @@ export class ScribeLambdaFactory
 		const lumberProperties = getLumberBaseProperties(documentId, tenantId);
 
 		try {
-			document = await this.documentRepository.readOne({ documentId, tenantId });
+			document = (await runWithRetry(
+				async () => this.documentRepository.readOne({ documentId, tenantId }),
+				"readIDocumentInScribeLambdaFactory",
+				6 /* maxRetries */,
+				30000 /* retryAfterMs */,
```

Comment:
Curious how we came up with this? Isn't this usually 3s? 30s seems like a lot. Also, the typical maxRetries seems to be 3 in session helped, checkpoint manager etc

Reply:
Made retries as 3, and interval as 1s. :)